vars/fortifyScan.groovy

```diff
@@ -0,0 +1,23 @@
+#!groovy
+import uk.gov.hmcts.contino.Builder
+import uk.gov.hmcts.contino.PipelineCallbacksRunner
+import uk.gov.hmcts.contino.AppPipelineConfig
+
+def call(params) {
+  PipelineCallbacksRunner pcr = params.pipelineCallbacksRunner
+  AppPipelineConfig config = params.appPipelineConfig
+  Builder builder = params.builder
+
+  def environment = params.environment
+  def product = params.product
+
+  if (config.fortifyScan) {
+    stageWithAgent("Fortify Scan", product) {
+      withFortifySecrets(product, environment) {
```

Comment:
This is for static analysis of source code which is non-environment specific, so why pass the environment in here? It may lead to confusion.

From what I can see the only reason to pass it in is to look up a key vault using a hardcoded vault naming convention. Better to just pass the specific vault name in here.